### PR TITLE
Recorded test for CheckConnectivity Api

### PR DESCRIPTION
**Pull request by irrogozh**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by irrogozh** [comment]
Link to the corresponding issue: Workitem for chackConnectivity API: https://github.com/Azure/azure-sdk-for-net/pull/3149 #3154

File: src/SDKs/Network/Network.Tests/Tests/CheckConnectivityTests.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using Microsoft.Azure.Management.Network;
+using Microsoft.Azure.Management.Network.Models;
+using Microsoft.Azure.Management.Resources;
```
Comment (shahabhijeet) [inline src/SDKs/Network/Network.Tests/Tests/CheckConnectivityTests.cs]: @irrogozh please use the latest Resource manager nuget package >=1.6.0 to record your tests

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/Network/Network.Tests/Tests/CheckConnectivityTests.cs (hunk 1 above)
Comment (irrogozh) [inline src/SDKs/Network/Network.Tests/Tests/CheckConnectivityTests.cs]: Abhijeet, I looked into Nuget gallery. https://www.nuget.org/packages/Microsoft.Azure.Management.ResourceManager/1.6.0-preview It seems like this version is the latest one...

Comment (irrogozh) [inline src/SDKs/Network/Network.Tests/Tests/CheckConnectivityTests.cs]: And I use this version already. Here is reference from Network.Tests.csproj
    <PackageReference Include="Microsoft.Azure.Management.ResourceManager" Version="1.6.0-preview" />

Comment (shahabhijeet) [inline src/SDKs/Network/Network.Tests/Tests/CheckConnectivityTests.cs]: @irrogozh yes you have the right references, but you need to create a ResourceManagment Client using the new nuget and you are still using the old client and recorded your test using the old client.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**